src/likelihoods/link.jl

```diff
@@ -0,0 +1,21 @@
+abstract type AbstractLink end
+
+(f::AbstractLink)(x) = apply(f, x)
+
+struct Link{F} <: AbstractLink
+    f::F
+end
+
```

Comment:
As mentioned before I think it is good to have a identity link for the sake of completeness.